src/App.js

```diff
@@ -1,16 +1,43 @@
 import React from 'react';
+//import { scryRenderedComponentsWithType } from 'react-dom/test-utils';
 import './App.css';
+//import ChatEntry from './components/ChatEntry';
+// import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
+import { useState } from 'react';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  const [likeData, setLikeData] = useState(chatMessages);
```

Comment:
Great use of the state variable! Well done recognizing we don't really need a state variable for the like counts!